### PR TITLE
:bug: Add JIRA status values

### LINKED ISSUE #974: Migration waves aggregated status value shows "In Progress" although there are no tickets
To reproduce, 

Create migration wave, add applications, export them to Jira instance.
Then remove tickets on Jira side.

### PR DESCRIPTION
**Pull request by ibolton336**
Closes #974 #911 

- Adds Completed status
- Adds In Progress status
- Filters out empty status from derived status calculation. This could mean the issue has been deleted from the underlying JIRA instance OR that the jira ticket is in the process of being created. We have no way of knowing how to interpret empty string, so this filters it out which gives us a more accurate representation of aggregated status.

### CODE REVIEW

File: client/src/app/utils/waves-selector.ts [hunk 1]
```diff
@@ -34,17 +34,12 @@ export const getWavesWithStatus = (
     const statuses = getTicketStatus(wave, tickets);
     if (statuses.includes("Error")) {
       return "Error";
-    } else if (statuses.includes("")) {
-      const now = dayjs.utc();
```
Comment (gildub) [inline client/src/app/utils/waves-selector.ts:38]: Are we not handling the starting date anymore for the "In progress" aggregate status value ?

**Review by gildub — COMMENTED** [review]
Just wondering if requirements have changed

**Comment by ibolton336** [comment]
> 

@gildub Ok understood. Wondering how we can best handle the empty ticket status false positive when issue has been deleted from the underlying JIRA instance.

**Comment by gildub** [comment]
@rromannissen, as of now when clicking on the aggregated status tab of a migration wave which has applications assigned to it, either because not exported yet or because the issues were removed in Jira, we're displaying an empty line.
Mabye we need to be provide more information to the user and display explicitly that there are no tickets.
What do you think ?

File: client/src/app/utils/waves-selector.ts (hunk 1 above)
Comment (ibolton336) [reply client/src/app/utils/waves-selector.ts:38]: I was thinking of this and wondering where the start date logic falls in with the other status we are deriving from JIRA. When would we prioritize using the date to derive status over using the status of the issues to derive status? Seems as if there should be a state for when all issues are in "New" or "Not started" state but the migration date is technically "In progress", for example.

**Comment by rromannissen** [comment]
@gildub sounds like a good idea to me! If I understood correctly, it's just about displaying a message saying something like "No issues created yet" when there are no issues (initial state) or all issues associated to the applications have been deleted. Is that right?

**Comment by gildub** [comment]
@rromannissen, that's right.

@ibolton336, let's check with UXD for the best way to present empty applications tickets to the user.
Maybe on a follow-up PR if easier.

**Comment by ibolton336** [comment]
Sounds great @gildub @rromannissen I will capture the new status proposition in an issue.

**Pull request closed without merging**